boa/environment.py

```diff
@@ -470,9 +470,22 @@ class FakeMessage(Message):  # Message object with settable attrs
         return self.vm.state.computation_class.apply_message(self.vm.state, msg, tx_ctx)
 
     # function to time travel
-    def time_travel(self, sleep_time):
-        self.vm.patch.timestamp += sleep_time
-        self.vm.patch.block_number += sleep_time // 12
+    def time_travel(
+        self,
+        seconds: Optional[int] = None,
+        blocks: Optional[int] = None,
+        block_delta: int = 12,
+    ):
+
+        if not (seconds or blocks):
```

Comment:
i think 0 value for seconds or blocks is ok, just like negative numbers. so i think a clean check here is `if seconds is None == blocks is None`

Reply:
```suggestion
        if seconds is None == blocks is None:
```